### PR TITLE
rework the wehauth doc

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Review the Aerohive webauth documentation

# Impacts
Doc

### CODE REVIEW

File: docs/networkdevice/aerohive.asciidoc [hunk 1]
```diff
@@ -177,32 +179,67 @@ Create a new Captive Portal Profile:
...
 * click on _Captive Web Portal Login Page Settings_ to deploy the configuration window
-* _Login URL_ must be *http://192.168.1.5/Aerohive::AP*
+* _Login URL_ must be *http://192.168.1.5/AeroHIVE::AP*
 * _Password Encryption_ : *No Encryption (Plaintext Password)*
 * click on *Save*
 
 image::docs/images/pfaerohive14.png[scaledwidth="100%",alt="Captive Portal SSID configuration"]
 
-Create or modify an SSID to add the Captive Portal fonction:
+Create a SSID to add the Captive Portal fonction:
```
Comment (cgx) [inline docs/networkdevice/aerohive.asciidoc]: I suggest *Create a SSID to enable Captive Portal functionality:*

File: docs/networkdevice/aerohive.asciidoc [hunk 2]
```diff
@@ -177,32 +179,67 @@ Create a new Captive Portal Profile:
...
-* from _HiveOS Number of devices running earlier versions of HiveOS_, select *Upgrade these devices to the latest version of HiveOS*
-* click on _Update_
-* wait until the date and time appears under _Update Status_
+* from the HiveManager, go to *CONFIGURATION -> NETWORK POLICIES* 
+* choose Network Policy and click OK, you should see this:
+
+image::docs/images/pfaerohive15.png[scaledwidth="100%",alt="External portal SSID configuration"]
+
+* under Authentication click on _<Captive Web Portal>_ and select the captive portal previously configured
+* once the _<RADIUS Settings>_ appaers under the captive portal, click on it
```
Comment (extrafu) [inline docs/networkdevice/aerohive.asciidoc]: appears

File: docs/networkdevice/aerohive.asciidoc [hunk 3]
```diff
@@ -177,32 +179,67 @@ Create a new Captive Portal Profile:
...
+* on that new windows _Choose RADIUS_ click New
```
Comment (extrafu) [inline docs/networkdevice/aerohive.asciidoc]: window

File: docs/networkdevice/aerohive.asciidoc [hunk 4]
```diff
@@ -177,32 +179,67 @@ Create a new Captive Portal Profile:
...
+Configure the User profile:
+
+image::docs/images/pfaerohive19.png[scaledwidth="100%",alt="CoA configuration"]
+
+* under _User Profile_ , click on *Add/Remove* and click on New
+
+image::docs/images/pfaerohive20.png[scaledwidth="100%",alt="User Profile configuration"]
+
+* enter the profile name, the VLAN ID and create the default VLAN as the same as the attribute number
+* create a new default vlan, click on the *+* button
```
Comment (extrafu) [inline docs/networkdevice/aerohive.asciidoc]: vlan -> VLAN

**Pull request merged**